.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "airbnb-base"
+}
```

Comment:
I don't think you need this file on this branch.

Reply:
Ok Godfrey, i am presently working on fixing it. Thank you